hedy.py

```diff
@@ -1496,6 +1498,8 @@ def assign_list_equals(self, args):
 
     def assign(self, args):
         right_hand_side = args[1]
+        left_hand_side = args[0]
+        self.check_var_usage([left_hand_side])
```

Comment:
I don't understand why we are checking the left-hand side here, shouldn't that be the right-hand side? That should exist so we can assign it?

Reply:
This is not needed either, assignments will be checked in type checker